emhttp/process_help.py

```
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import os
+import re
+import glob
+
+def read_helptext_sections(helptext_file):
+    sections = {}
+    current_section = None
+    current_content = []
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)